src/components/00-design-tokens/_breakpoints.scss

```diff
@@ -0,0 +1,5 @@
+$bp-tiny: 375px;
+$bp-small: 700px;
+$bp-narrow: 769px;
+$bp-medium: 1024px;
+$bp-wide: 1280px;
```

Comment:
If we want to namespace these, I feel like `mq-` (media query) might be a more intuitive prefix?